README.md

```diff
@@ -24,7 +24,8 @@ const combinedComposables = composable0.compose(composable1, composable2, compos
 
 ### Descriptor
 
-**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain the information necessary to create an object instance by a *composable*.
+**Composable descriptor** (or just **Descriptor**) is a meta data object. It's a collection of properties that contain
+the information necessary to create an object instance by a *composable*.
```

Comment:
From my team. Only 1 of 9 people is using soft wraps.

Also, many code linters disallow too long lines by default: https://gist.github.com/nkbt/9efd4facb391edbf8048#file-eslintrc-js-L216